vanilla.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    let page = document.getElementById("page");
+    let form = document.querySelector("form");
+
+    // ^ get the nodes we will be manipulating
+
+    form.addEventListener("submit", addPost)
+
+    // ^ adding event listener and then prepending the post on submit
+
+});
+
+function addPost(event){
+    event.preventDefault();
+
+    // ^ prevent page refresh so we can keep the post appended!
+
+    let newPost = document.createElement("div");
+    let input = document.querySelector("input");
+
+    // ^ have to get the input node here so we have the input value at time of submit
+
+    newPost.textContent = input.value + " - " + returnSeconds();
+    page.prepend(newPost);
```

Comment:
Will this create a global variable or does this function somehow have access to the binding scope?

Reply:
Are you referring to the "page" variable here?